lib/brainstem/query_strategies/filter_or_search.rb

```diff
@@ -67,7 +63,7 @@ def run_search(scope, includes, sort_name, direction)
         end
       end
 
-      def paginate(scope)
+      def paginated_scopes(scope)
```

Comment:
Similarly it seems like this should be the responsibility of the Paginator not the Filterer or the Searcher.